packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/FGMImageRegistry.h

```diff
@@ -0,0 +1,21 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Flutter/Flutter.h>
+#import <GoogleMaps/GoogleMaps.h>
+
```

Comment:
For new code in this repo `NS_ASSUME_NONNULL_*` should be used in headers. You can check other headers for reference.